bootstrap-salt.sh

```diff
@@ -549,10 +549,17 @@ if [ "$ITYPE" != "git" ]; then
     fi
 fi
 
-# Check for -r if -R is being passed. Set -r with a warning.
-if [ "$_CUSTOM_REPO_URL" != "null" ] && [ "$_DISABLE_REPOS" -eq $BS_FALSE ]; then
-    echowarn "Detected -R option. No other repositories will be configured when -R is used. Setting -r option to True."
-    _DISABLE_REPOS=$BS_TRUE
+# Set the _REPO_URL value based on if -R was passed or not. Defaults to repo.saltstack.com.
+if [ "$_CUSTOM_REPO_URL" != "null" ]; then
+    _REPO_URL="$_CUSTOM_REPO_URL"
+
+    # Check for -r since -R is being passed. Set -r with a warning.
+    if [ "$_DISABLE_REPOS" -eq $BS_FALSE ]; then
+        echowarn "Detected -R option. No other repositories will be configured when -R is used. Setting -r option to True."
+        _DISABLE_REPOS=$BS_TRUE
+    fi
+else
+    _REPO_URL="repo.saltstack.com"
```

Comment:
I think, maybe it's better to put the default `_REPO_URL` value even earlier, where all default settings already defined?
And then just override it here. Just to keep all similar things in one place.

See line 194: https://github.com/saltstack/salt-bootstrap/pull/1045/files#diff-a8778033f6225c137abf05cf8d0bb113L194